radix-engine-common/src/native_addresses.rs

```diff
@@ -141,6 +141,13 @@ pub const ACCESS_CONTROLLER_PACKAGE: PackageAddress = PackageAddress::new_or_pan
     60, 165, 22, 134, 49, 140, 99, 24, 198,
 ]);
 
+/// The native package for pools.
+pub const POOL_PACKAGE: PackageAddress = PackageAddress::new_or_panic([
```

Comment:
Will need a new package address here. @dhedey is this something your script can provide?  We do not have one for the pool package in REP 71.